apps/studio/components/layouts/ProjectLayout/RestoreFailedState.tsx

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link'
+import { Button } from 'ui'
+import { CriticalIcon } from 'ui-patterns/Icons/StatusIcons'
+
+import { useProjectContext } from './ProjectContext'
+
+const RestoreFailedState = () => {
+  const { project } = useProjectContext()
+
+  return (
+    <div className="flex items-center justify-center h-full">
+      <div className="bg-surface-100 border border-overlay rounded-md w-3/4 lg:w-1/2">
+        <div className="space-y-6 pt-6">
+          <div className="flex px-8 space-x-8">
+            <div className="mt-1">
+              <CriticalIcon className="w-5 h-5" />
+            </div>
+            <div className="space-y-1">
+              <p>Something went wrong while restoring your project</p>
```

Comment:
maybe lets add some comforting language haha

"Don't worry! All of your project's data is still intact, but your project is inaccessible due to the restoration failure. Please contact support for assistance."